### PR TITLE
NMS-13472: update table formatting in docs

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ ] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ ] Have you made a comment in that issue which points back to this PR?
* [ ] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13472

### CODE REVIEW

**Comment by Bonrob2** [comment]
@mmahacek , note that the ticketing sections (Jira and Remedy) refer to supported versions. I was unable to update ... Chandra is going to investigate next week, so we can update it once we have the info.

File: docs/modules/operation/pages/operation/newts/newts-repository-converter.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-= Newts Repository Converter: RRD/JRB to Newts migration utility
+= Newts Repository Converter: RRD/JRB to Newts Migration Utility
```
Comment (mmahacek) [inline docs/modules/operation/pages/operation/newts/newts-repository-converter.adoc:2]:
```suggestion
RRD/JRB to Newts Migration Utility
```

File: docs/modules/operation/pages/operation/ssl/ssl.adoc [hunk 2]
```diff
@@ -37,20 +37,20 @@ keytool \
...
+If you updated an existing Java Truststore, type the password you chose when you first created the Java Truststore.
 
 [[ga-operation-download-certificate]]
 === Download existing public certificate
 
-To Download an existing public certificate the following command can be issued.
+Use the following command to download an existing public certificate.
 
 .Download an existing public certificate
 [source,shell]
```
Comment (mmahacek) [inline docs/modules/operation/pages/operation/ssl/ssl.adoc:56]:
```suggestion
[source, console]
```

File: docs/modules/operation/pages/operation/ssl/ssl.adoc [hunk 3]
```diff
@@ -64,58 +64,59 @@ openssl \
...
 === Configure {page-component-title} to use the defined Java Truststore
 
-To setup {page-component-title} to use the defined Java Truststore the according `javax.net.ssl.trustStore*` properties have to be set.
-Open `$\{OPENNMS_HOME}/etc/opennms.properties` and add the properties `javax.net.ssl.trustStore` and `javax.net.ssl.trustStorePassword` as shown below.
+To set up {page-component-title} to use the defined Java Truststore you must set the applicable `javax.net.ssl.trustStore*` properties.
+Open `$OPENNMS_HOME/etc/opennms.properties` and add the properties `javax.net.ssl.trustStore` and `javax.net.ssl.trustStorePassword` as shown below.
 
-.$\{OPENNMS_HOME}/etc/opennms.properties snippet to define a Java Truststore
+.$OPENNMS_HOME/etc/opennms.properties snippet to define a Java Truststore
 [source]
```
Comment (mmahacek) [inline docs/modules/operation/pages/operation/ssl/ssl.adoc:80]:
```suggestion
[source, properties]
```

File: docs/modules/operation/pages/operation/ssl/ssl.adoc [hunk 4]
```diff
@@ -64,58 +64,59 @@ openssl \
...
 ----
 javax.net.ssl.trustStore=/$\{OPENNMS_HOME}/etc/trust-store.jks <1>
```
Comment (mmahacek) [inline docs/modules/operation/pages/operation/ssl/ssl.adoc:82]:
```suggestion
javax.net.ssl.trustStore=$OPENNMS_HOME/etc/trust-store.jks <1>
```

File: docs/modules/operation/pages/operation/timeseries/configuration.adoc [hunk 5]
```diff
@@ -3,7 +3,7 @@
 
 == Enabling Time Series Integration Layer
 
-{page-component-title} can be configured to use the Time Series Integration Layer by setting the following property in `$\{OPENNMS_HOME}/etc/opennms.properties`:
+{page-component-title} can be configured to use the Time Series Integration Layer by setting the following property in `$OPENNMS_HOME/etc/opennms.properties`:
 
 [source]
```
Comment (mmahacek) [inline docs/modules/operation/pages/operation/timeseries/configuration.adoc:8]:
```suggestion
[source, properties]
```

File: docs/modules/operation/pages/operation/newts/newts-repository-converter.adoc [hunk 6]
```diff
@@ -31,40 +31,41 @@ Finally, we convert and persist the Rrd- or JRobin-based data to Newts using the
...
 |===
 
 == Example 1: Convert RRD-based data with storeByGroup enabled
 
 The following example shows how to convert RRDTool-based data that was stored with `storeByGroup` enabled.
-The {page-component-title} home is `/opt/opennms`, the data directory is `/opt/opennms/share/rrd`, and the RRDTool binary located at `/usr/local/bin/rrdtool`.
+The {page-component-title} home is `/opt/opennms`, the data directory is `/opt/opennms/share/rrd`, and the RRDTool binary is located at `/usr/local/bin/rrdtool`.
 This program call uses 16 concurrent threads to convert the RRD files.
 
 [source, shell]
```
Comment (mmahacek) [inline docs/modules/operation/pages/operation/newts/newts-repository-converter.adoc:71]:
```suggestion
[source, console]
```

File: docs/modules/operation/pages/operation/newts/newts-repository-converter.adoc [hunk 7]
```diff
@@ -31,40 +31,41 @@ Finally, we convert and persist the Rrd- or JRobin-based data to Newts using the
...
 
-IMPORTANT: The _newts_converter_ utility needs the path to the base directory of your {page-component-title} instance to read the configuration files.
+IMPORTANT: The newts_converter utility needs the path to the base directory of your {page-component-title} instance to read the configuration files.
 For example, the utility needs the datasource configuration during the migration process to query the database to look up node data.
 
 == Use
-The utility is installed by default and its wrapper script is located in the `$\{OPENNMS_HOME}/bin` directory.
+The utility is installed by default and its wrapper script is located in the `$OPENNMS_HOME/bin` directory.
 
 [source, shell]
```
Comment (mmahacek) [inline docs/modules/operation/pages/operation/newts/newts-repository-converter.adoc:42]:
```suggestion
[source, console]
```

File: docs/modules/operation/pages/telemetryd/introduction.adoc [hunk 8]
```diff
@@ -84,7 +85,7 @@ For example:
 </queue>
 ----
 
-When setting these options on Minion, you can add them as `parser` properties, and on Sentinel as `adapter` properties:
+When setting these options on Minion, you can add them as parser properties, and on Sentinel as adapter properties:
 [source]
```
Comment (mmahacek) [inline docs/modules/operation/pages/telemetryd/introduction.adoc:89]:
```suggestion
[source, properties]
```

File: docs/modules/operation/pages/ticketing/jira.adoc [hunk 9]
```diff
@@ -7,46 +7,47 @@ The Jira ticketing plugin creates JIRA issues in response to {page-component-tit
...
 == Setup
 
-First, you'll need to install the `opennms-plugin-ticketer-jira` package for your system.
+First, you need to install the `opennms-plugin-ticketer-jira` package for your system.
 The Jira ticketing plugin and its dependencies are not part of the core packages.
 
-To enable the plugin, start by setting following property in `$\{OPENNMS_HOME}/etc/opennms.properties`:
+To enable the plugin, start by setting following property in `$OPENNMS_HOME/etc/opennms.properties`:
 
 [source]
```
Comment (mmahacek) [inline docs/modules/operation/pages/ticketing/jira.adoc:15]:
```suggestion
[source, properties]
```

File: docs/modules/operation/pages/ticketing/jira.adoc [hunk 10]
```diff
@@ -7,46 +7,47 @@ The Jira ticketing plugin creates JIRA issues in response to {page-component-tit
...
 
-Next, add `jira-troubleticketer` to the `featuresBoot` property in the `$\{OPENNMS_HOME}/etc/org.apache.karaf.features.cfg`
+Next, add `jira-troubleticketer` to the `featuresBoot` property in the `$OPENNMS_HOME/etc/org.apache.karaf.features.cfg`.
 
 Restart {page-component-title}.
 
-When {page-component-title} has started again, login to the Karaf Shell and install the feature:
+When {page-component-title} has started, log in to the Karaf shell and install the feature:
 
 [source]
```
Comment (mmahacek) [inline docs/modules/operation/pages/ticketing/jira.adoc:52]:
```suggestion
[source, console]
```

File: docs/modules/operation/pages/ticketing/jira.adoc [hunk 11]
```diff
@@ -55,41 +56,41 @@ feature:install jira-troubleticketer
...
+Besides the common fields creator, create date, description, or subject, you also usually need to set ticket-system proprietary fields.
 
 In some cases, even so-called additional custom fields are defined.
 
 To set these fields, the Jira ticketing plugin provides the opportunity to define those in the OpenNMS ticket attributes, which can be overwritten with Drools use.
 
-To enable the Drools ticketing integration, set the following property in `$\{OPENNMS_HOME}/etc/opennms.properties`:
+To enable the Drools ticketing integration, set the following property in `$OPENNMS_HOME/etc/opennms.properties`:
 
 [source]
```
Comment (mmahacek) [inline docs/modules/operation/pages/ticketing/jira.adoc:81]:
```suggestion
[source, properties]
```

File: docs/modules/operation/pages/ticketing/jira.adoc [hunk 12]
```diff
@@ -55,41 +56,41 @@ feature:install jira-troubleticketer
...
 ----
 opennms.ticketer.servicelayer=org.opennms.netmgt.ticketd.DroolsTicketerServiceLayer
 ----
 
-In addition, the property in `$\{OPENNMS_HOME/etc/drools-ticketer.properties` must point to a `drools-ticketer-rules.drl` file:
+In addition, the property in `$OPENNMS_HOME/etc/drools-ticketer.properties` must point to a `drools-ticketer-rules.drl` file:
 
 [source]
```
Comment (mmahacek) [inline docs/modules/operation/pages/ticketing/jira.adoc:88]:
```suggestion
[source, properties]
```

File: docs/modules/operation/pages/ticketing/jira.adoc [hunk 13]
```diff
@@ -55,41 +56,41 @@ feature:install jira-troubleticketer
...
 ----
 drools-ticketer.rules-file=$\{OPENNMS_HOME/etc/drools-ticketer-rules.drl
```
Comment (mmahacek) [inline docs/modules/operation/pages/ticketing/jira.adoc:90]:
```suggestion
drools-ticketer.rules-file=$OPENNMS_HOME/etc/drools-ticketer-rules.drl
```

File: docs/modules/operation/pages/ticketing/remedy.adoc [hunk 14]
```diff
@@ -28,7 +29,7 @@ Currently supported Remedy product versions are listed below:
 
 The Remedy ticketing plugin and its dependencies are part of the {page-component-title} core packages.
 
-Start by enabling the plugin and the ticket controls in the {page-component-title} web interface, by setting the following properties in `$\{OPENNMS_HOME}/etc/opennms.properties`:
+Start by enabling the plugin and the ticket controls in the {page-component-title} web interface, by setting the following properties in `$OPENNMS_HOME/etc/opennms.properties`:
 
 [source]
```
Comment (mmahacek) [inline docs/modules/operation/pages/ticketing/remedy.adoc:34]:
```suggestion
[source, properties]
```

**Review by mmahacek — APPROVED** [review]
Looks good

**Pull request merged**